src/main/java/bob/Storage.java

```diff
@@ -0,0 +1,69 @@
+package bob;
+
+import bob.Task.Task;
+import bob.command.ListCommand;
+import bob.exception.FileException;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.List;

```

Comment:
Files are all properly packaged, imports are neatly listed out, same goes for all other classes 👍